packages/material-ui-lab/src/MasonryItem/MasonryItem.test.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { createClientRender, describeConformanceV5 } from 'test/utils';
+import MasonryItem, { masonryItemClasses as classes } from '@material-ui/lab/MasonryItem';
+import { expect } from 'chai';
+import { createTheme } from '@material-ui/core/styles';
+import defaultTheme from '@material-ui/core/styles/defaultTheme';
+import ResizeObserver from 'resize-observer-polyfill';
+import { style } from './MasonryItem';
```

Comment:
All of the tests using this should probably be visual regression tests. The unit tests look like they will fail every time we change the implementation slightly.